.github/CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+# CODEOWNERS file
```

Comment:
maybe worth just adding a line or reference, what is the aim/role of codeowners? cause it would mean that the files/folders listed here, if modified, won't be able to be merged unless approval for code owners?